enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -0,0 +1,310 @@
+---
+title: cluster-logging-log-forwarding
+authors:
+  - "@jcantrill"
+reviewers:
+  - "@bparees"
+  - "@ewolinetz"
+  - "@jeremyeder"
+approvers:
+  - TBD
+creation-date: 2019-09-17
+last-updated: 2019-09-17
+status: implementable
+see-also:[]
+replaces:[]
+superseded-by:[]
+---
+
+# cluster-logging-log-forwarding
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+The purpose of log forwarding is to provide a declarative way by which adopters of cluster logging can ship
+container and node logs to destinations that are not necessarily managed by the OKD cluster logging infrastructure.
+Destinations are either on or off cluster endpoints such as the cluster logging provided Elasticsearch, an organization's Kafka message bus, or other log aggregation solutions (e.g. Splunk, Graylog).
+
+## Motivation
+
+Organizations desire to reuse their existing enterprise log solutions to store container logs.  Providing a declarative mechanism by which administrators define a log destination simplifies their operational burden.  The cluster logging stack is able to deploy a collector to each OKD node that is configured with the necessary permissions to collect the logs, add container metadata (e.g. labels) and ship them to the specified endpoint.
+
+
+
+### Goals
+The specific goals of this proposal are:
+
+* Provide an API that identifies log sources and their desired destination
+* Send logs to an Elasticsearch cluster not managed by the cluster logging infrastructure
+* Send logs to a fluentd instance not managed by the cluster logging infrastructure via fluentd's secureforward protocol
```

Comment:
any other targets that should/must be supported?

Reply:
> any other targets that should/must be supported?

I explicitly only chose to mention the ones we discussed supporting in 4.3.  Do we need to mention others at this time?

Reply:
I explicitly chose not to mention them here as we are not putting them into 4.3.  Should we mention what may be included in the future?

Reply:
nope..just wanted to make sure what you listed was (currently) exhaustive.